_posts/2.4/2021-06-09-new.md

```diff
@@ -381,3 +383,27 @@ The best ways to contribute at the current time are:
 - Try out installing BigBlueButton 2.4 and see if you spot any issues
 - Help test a [2.4 pull request](https://github.com/bigbluebutton/bigbluebutton/pulls?q=is%3Aopen+is%3Apr+milestone%3A%22Release+2.4%22) in your development environment
   <!-- TODO create a GitHub label for contributions-welcome and link here -->
+
+## API
+
+### Deprecation of password
+
+Even though it is still possible to create a meeting with a password, it is now being deprecated, and it is not required anymore. So here are some explanations to better ilustrate this update:
+
+- **Create** - It is possible and recommended to not use the passwords while creating the request:
+
+```url 
+https://bbb25.bbbvm.imdt.com.br/bigbluebutton/api/create?allowStartStopRecording=true&attendeePW=ap&moderatorPW=mp&... &checksum=
+```
+or 
+```url 
+https://bbb25.bbbvm.imdt.com.br/bigbluebutton/api/create?allowStartStopRecording=true&... &checksum=
+```
+
+- **Join** - These requests don't need to follow the `/create` endpoint, on the grounds that it has its own parameter to know the role of the user. Se the explanation down below:
+
+  - In any cases, if the `role` parameter is sent along, it will be respected, even if `password` is passed too;
+  - If the `role` parameter is not passed in the call, and the password exists (i.e., it has been sent in `/create` endpoint), then the password is mandatory for the `/join` endpoint to acknowledge which role does the user have.
+  - If the password does not exist (i.e., it was not defined in `/create` endpoint), then the `role` parameter is mandatory, otherwise it is not possible to enter a meeting.
+
+- **End** - the password is really not used here. But, if it was set (in the `/create` endpoint) and passed along in the `end` endpoint, it will be validated, otherwise, it won't.
```

Comment:
```suggestion
- **End** - the password is really not used here. But, if it was set (in the `/create` endpoint) and provided along in the `end` endpoint, it will be validated, otherwise, it won't.
```